src/content/tutorials/en/intro-to-p5.spline.mdx

```diff
@@ -0,0 +1,538 @@
+---
```

Comment:
Maybe we can rename the file to something like `into-to-splines` since it's about splines in general than of the single `spline()` function?